src/bokeh/models/plots.py

```diff
@@ -301,6 +306,11 @@ def add_layout(self, obj: Renderer, place: PlaceType = "center") -> None:
                 f"Invalid place '{place}' specified. Valid place values are: {nice_join(Place)}",
             )
 
+        for name in Place:
+            panel = getattr(self, name)
+            if obj in panel:
+                panel.remove(obj)
```

Comment:
```suggestion
            while obj in panel:
                panel.remove(obj)
```

Not sure if this ever happens, but it could give a problem if the object is twice in the panel.

Reply:
It's a good idea to be thorough. On a side note, for a "batteries included" branded language could provided use with `remove_all()`. I guess I should have used a filtering comprehension instead.